source/sourcing_and_processing_data/sourcing_data.rst

```diff
@@ -0,0 +1,126 @@
+Sourcing data
+=============
+
+There are 3 different methods to add data to a dataset:
+
+* by uploading a static, local file into the platform
+* by connecting the platform to a remote service (e.g. an API, a geographical information system, an FTP, etc.): it enables the platform to retrieve the files that the remote service contains while staying synchronized with it
```

Comment:
"the files that the remote" => "data that the remote ..."
Api are not really files.